### DISCUSSION TITLE
Add *co-ordinates* to the Crosshair mode.

### DISCUSSION THREAD

**Discussion by CaptainCooky** [Mojang/minecraft-editor#54] [Feedback]
Adding the coordinates to the lower left corner of the Crosshair Mode would be appreciated. They are in the Tool Mode, and it would be nice if they were also in the Crosshair Mode. We could, of course, enable the *gamerule* `showcoordinates=true`. However, they crowd the screen when back in Tool Mode. 
A minor request, to be sure, but it'd be nice to have! :D

**Comment by minecraftchloe** [comment]
Going to review technically how we might do this in a non-obstructive way in the UI!
